packages/flutter_splash_tizen/example/tizen/tizen-manifest.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest package="com.example.flutter_splash_tizen_example" version="1.0.0" api-version="4.0" xmlns="http://tizen.org/ns/packages">
+    <profile name="wearable"/>
+    <ui-application appid="com.example.flutter_splash_tizen_example" exec="Runner.dll" type="dotnet" multiple="false" nodisplay="false" taskmanage="true" api-version="4">
+        <label>flutter_splash_tizen_example</label>
+        <icon>ic_launcher.png</icon>
+        <metadata key="http://tizen.org/metadata/prefer_dotnet_aot" value="true"/>
+    </ui-application>
+    <tv-info api-version="8.8.0">
```

Comment:
The example app is expected to work out of the box without requiring additional actions from users. Please add the `<splash-screens>` section to tizen-manifest.xml.